### ISSUE TITLE
Detect external changes of tilesets

### ISSUE THREAD

**Issue by issy123** [bjorn/tiled#1785]
Tiled does detect external changes of the .tmx files but would be nice if it also detects changes on the .tsx files and auto reload.

**Comment by bjorn** [comment]
Hmm, it only does that when the tileset is explicitly opened, rather than just being loaded as a dependency of a map. I agree it should be extended to reload as well in the latter case.

**Comment by UliAbo** [comment]
> Hmm, it only does that when the tileset is explicitly opened, rather than just being loaded as a dependency of a map. I agree it should be extended to reload as well in the latter case.

Yes it would be great to have it auto-reloaded without it being opened.
